src/docfx/resolve/ResolveContent.cs

```diff
@@ -6,5 +6,6 @@
 
 namespace Microsoft.Docs.Build
 {
-    internal delegate (string content, Document file) ResolveContent(Document relativeTo, string href);
+    // todo: insert to markdown pipleline to generate inclusion mappings
```

Comment:
so for TOC, `ResolveContent` has two purposes:
1. embed nested TOC
2. peek content to get first link

This is different from markdown. Suggest to use different delegates instead of share one.

Reply:
see todo, it is the same purpose when markdown pipeline use this delegate to resolve inclusion content

---
In reply to: [193351668](https://github.com/dotnet/docfx/pull/2851#discussion_r193351668) [](ancestors = 193351668)